action.yml

```diff
@@ -8,7 +8,7 @@ inputs:
   actions:
     description: "Actions to correct"
     required: false
-    default: ".github/workflows"
+    default: '[".github/workflows"]'
```

Comment:
Hey @segiddins thank you for your contributio!
I'm not an expert here, but this looks like a breaking change, how would this impact existing users of the action?
cc @evankanderson